internal/deb/extract_test.go

```diff
@@ -283,6 +284,10 @@ var extractTests = []extractTest{{
 }}
 
 func (s *S) TestExtract(c *C) {
+	oldUmask := syscall.Umask(02)
+	defer func() {
+		syscall.Umask(oldUmask)
```

Comment:
These changes look unrelated to this PR. What is this about?  In theory extraction should work the same independently from whether umasks are set or not. If that's not the case, that's a bug we'll want to see fixed.

In either case, can we please move this out into its own PR?

Reply:
Created https://github.com/canonical/chisel/pull/32 instead?